cpu/esp32/include/periph_cpu.h

```diff
@@ -20,6 +20,7 @@
 #define PERIPH_CPU_H
 
 #include <stdint.h>
+#include "sdk_conf.h"
```

Comment:
this wasn't here before?

Reply:
on esp32, `CLOCK_CORECLOCK` is defined in `sdk_conf.h`. If it's not included here, it doesn't build